srcopsmetrics/kebechet_metrics.py

```diff
@@ -150,6 +153,9 @@ def get_overall_stats_update_manager(self):
         """Return stats over whole repository age."""
         prs = self._get_update_manager_pull_requests()
 
+        if not prs:
+            return []
```

Comment:
it should be a dict according to the output right? Also below. You should be consistent, maybe also adding type type for the output of the methods? wdyt?

Reply:
yes man, you're absolutely right. Type hints added and changed to `dict`